spec/carrierwave/storage/aws_file_spec.rb

```diff
@@ -44,6 +44,22 @@
     end
   end
 
+  describe '#read' do
+    let(:s3_object) { double('Aws::S3::Object') }
```

Comment:
It would be nice to use an `instance_double` for this, so that we get api verification when `Aws::S3::Object` is loaded.